docs/cqrs/command/index.md

```diff
@@ -0,0 +1,98 @@
+# Command
+
+Command is just a class that implements the `ICommand` interface. Commands are used to execute an action that modifies data. Commands are validated, and if they pass validation, they should succeed. Commands do not return any value. This makes them quite constrained, yet reasoning is much easier.
+
+## Packages
+
+| Package | Link | Application in section |
+| --- | ----------- | ----------- |
+| LeanCode.Contracts | [![NuGet version (LeanCode.Contracts)](https://img.shields.io/nuget/vpre/LeanCode.Contracts.svg?style=flat-square&logo=nuget)](https://www.nuget.org/packages/LeanCode.Contracts/2.0.0-preview.3/) | `ICommand` |
+| LeanCode.CQRS.Execution | [![NuGet version (LeanCode.CQRS.Execution)](https://img.shields.io/nuget/vpre/LeanCode.CQRS.Execution.svg?style=flat-square&logo=nuget)](https://www.nuget.org/packages/LeanCode.CQRS.Execution/8.0.2260-preview/) | `ICommandHandler` |
+
+## Contract
+
+Consider the command that updates name of the `Project` (caller of the command is required to have `Employee` role and own project):
+
+```csharp
+namespace ExampleApp.Contracts.Projects;
+
+[ProjectIsOwned]
+[AuthorizeWhenHasAnyOf(Auth.Roles.Employee)]
+public class UpdateProjectName : ICommand, IProjectRelated
+{
+    public string ProjectId { get; set; }
+    public string Name { get; set; }
+
+    public static class ErrorCodes
+    {
+        public const int InvalidName = 1;
+        public const int ProjectIdInvalid = 2;
+        public const int ProjectDoesNotExist = 3;
+    }
+}
+```
+
+!!! tip
+    More on authorization and permissions can be found [here](../authorization/index.md) and on error codes and validation [here](../validation/index.md).
+
+## Naming conventions
+
+Commands in CQRS are responsible for modifying the state of the system, and therefore their names should reflect the action they trigger. A common convention is to use imperative verbs that clearly describe the intent, including the namespace as part of the contract, such as:
```

Comment:
This was @tomaszlaskowskiLC suggestion somewhere above. Namespace might sometimes provide additional information about the contract, and I think it should be treated as a part of it so that we do not create some weird folder structure which might confuse the clients.